.github/workflows/ci.yml

```diff
@@ -37,3 +37,9 @@ jobs:
         if: ${{ matrix.lint }}
 
       - run: mix test
+
+      - run: mix kudos.generate
+
+      - run: mix kudos.generate --dry-run
+
+      - run: mix kudos.generate --include-dev-deps
```

Comment:
This could be a future improvement: What do you think about committing the licensed.md file and asserting that it matches to what this command generates.

Submitting the license file is also a nice way to show visitors an example of the generated file

Reply:
Like the idea of committing the `licenses.md` unfortunately for non dev it would be empty.

Or what about a checker wich can be called in the pipeline, like `mix kudos.compliant` or `mix kudos.complete`. Will merge this for now. We can think about adding this in the next release.